lib/nice_partials/partial/content.rb

```diff
@@ -35,7 +36,7 @@ def write(...)
   private
 
   def process_options(new_options)
-    @options ||= Options.new(@view_context) and @options.merge!(new_options) unless new_options.empty?
+    @options.merge!(new_options) unless new_options.empty?
```

Comment:
I wonder if the `unless` is worth it now? `merge!` might do the same check in its C code? Then we could potentially inline `process_options` too.